samples/modelbuilder/ObjectDetection_StopSigns/StopSignDetection/README.md

```diff
@@ -0,0 +1,40 @@
+﻿---
```

Comment:
There is an invisible BOM character here that seems to prevent this from rendering correctly.

Before:

![image](https://user-images.githubusercontent.com/31348972/111354951-8a14db00-868f-11eb-9e7a-29774e658fb5.png)


After:

![image](https://user-images.githubusercontent.com/31348972/111354995-94cf7000-868f-11eb-94bf-7b0eaea67334.png)

This may prevent it from being recognized by @VSC-Service-Account

```suggestion
---
```

Reply:
It doesn't let me commit your suggestion since there's no recognizable changes.

Does the latest commit fix it?

Reply:
@briacht Nope. https://github.com/briacht/machinelearning-samples/pull/1 should do it.

Reply:
Once his PR is merged into your code @briacht I'll go ahead and approve.